doc/package.json

```diff
@@ -30,7 +31,8 @@
     "@libp2p/tcp": "^9.0.0",
     "aegir": "^41.0.2",
     "libp2p": "^1.0.0",
-    "prom-client": "^15.0.0"
+    "prom-client": "^15.0.0",
+    "typescript": "5.2.2"
```

Comment:
Let's wait for the fix to bubble up instead of trying to manage multiple versions of typescript in the monorepo.